wear/src/main/java/chat/rocket/android/util/TimberLogger.kt

```diff
@@ -0,0 +1,19 @@
+package chat.rocket.android.util
+
+import chat.rocket.common.util.PlatformLogger
+import timber.log.Timber
+
+object TimberLogger : PlatformLogger {
```

Comment:
Duplicated. Please remove. You can put this class in the `util` module and use it from there on the `wear` and `app` module as well.